server/util-server.js

```diff
@@ -413,22 +413,25 @@ exports.radius = function (
 exports.redisPingAsync = function (dsn) {
     return new Promise((resolve, reject) => {
         const client = redis.createClient({
-            url: dsn,
+            url: dsn
         });
         client.on("error", (err) => {
+            client.disconnect();
```

Comment:
The disconnect doesn't trigger an error event so there isn't a worry about an infinity loop.
I don't believe it's possible to get to this point and the isOpen not be true, but since the first thing the disconnect does is check isOpen and throws an exception, gating will be cleaner